docs/installguide/install_all.rst

```diff
@@ -54,23 +51,11 @@ _______
 
 To upgrade an existing KA Lite installation.
 
-#. Download the KA Lite `OSX installer <https://learningequality.org/r/osx-installer-0-15>`_.
-#. After the download is complete, double click the .dmg file.
-#. Click the ``Agree`` button to accept the LICENSE agreement.
-#. On the .dmg window, drag the ``KA-Lite Monitor`` app into the ``Applications`` folder.
-#. Give confirmation to overwrite the existing app.
-#. Launch ``KA-Lite Monitor`` from your ``Applications`` folder.
-
-.. tip::
-    If the sidebar shows entries that are greyed-out, the child items of the entry may be videos that were not yet downloaded.  If there are assessment items inside, then you need to extract the `assessment.zip` manually:
-
-    #. Launch ``KA-Lite Monitor`` from your ``Applications`` folder.
-    #. Click on the app icon at the menu bar.
-    #. Click on ``Preferences`` in the menu option.
-    #. Go to ``Advanced`` tab, click on the ``Extract Assessment`` button, then confirm the action. 
-    #. Restart the server, login to the web app, then check the sidebar contents.
-
-    This will enable the greyed-out assessment items on the sidebar.
+#. Download the KA Lite `OSX installer <https://learningequality.org/r/osx-installer-0-16>`_.
+#. Make sure that you stop the server and quit the KA Lite Monitor.
+#. After the download is complete, double click the .pkg file.
+#. Click the ``Agree`` button to accept the LICENSE agreement and follow the installer instructions.
+#. Launch ``KA-Lite app`` from your ``Applications`` folder.

```

Comment:
I think the same changes applies as per my comment above.  Let's replace the "Click the Agree button" and the "Launch KA-Lite app" steps with the steps I outlined above.